src/topics/data.js

```diff
@@ -100,6 +106,7 @@ function modifyTopic(topic, fields) {
 
 	if (topic.hasOwnProperty('timestamp')) {
 		topic.timestampISO = utils.toISOString(topic.timestamp);
+		topic.scheduled = topic.timestamp > Date.now();
```

Comment:
Can change this to 
```
if (!fields.length || fields.includes('scheduled')) {
  topic.scheduled = topic.timestamp > Date.now();
}
```
So that the `scheduled` field isn't returned if not requested.

Reply:
Done.